docs/recipes/schema-directives.md

```diff
@@ -47,20 +47,22 @@ const schemaDirectives = {
   date: FormattableDateDirective
 };
 
-const { schema } = new GraphQLModule({
+const CommonModule = new GraphQLModule({
   typeDefs,
   resolvers,
   schemaDirectives,
 });
 ```
 
-GraphQL Modules won't automatically apply your directives when it generates a schema for a module to avoid applying the same directive more than once. However, you can set the `visitSchemaDirectives` option to `true` in your root module and the directives from all your combined modules will be applied to your final schema.
+GraphQL Modules won't automatically apply your directives when it generates a schema for a module to avoid applying the same directive more than once. However, you can set the `visitSchemaDirectives` option to `true` in your **root module** and the directives from all your combined modules will be applied to your final schema.
```

Comment:
this might be why my unit tests fail.. It doesn't find the directives, in my error.